bin/specextract

```diff
@@ -1786,6 +1790,9 @@ def get_par(args):
     #pars["wmap_clip"] = params["wmap_clip"] = paramio.pgetstr(pfile,"wmap_clip")
     #pars["wmap_threshold"] = params["wmap_threshold"] = paramio.pgetstr(pfile,"wmap_threshold")
 
+    pars["parallel"] = params["parallel"] = paramio.pgetstr(pfile, "parallel")
+    pars["nproc"] = params["nproc"] = paramio.pgetstr(pfile, "nproc")
+
```

Comment:
NOT FOR THIS PR, but I would really like to some day replace the strings in the parameter values with Python-appropriate data types - e.g. for `parallel` use bool and `nproc` use int.